flake.nix

```diff
@@ -3,6 +3,8 @@
 
   inputs.nixpkgs.url = "github:NixOS/nixpkgs/nixpkgs-unstable";
 
+  nixConfig.substituters = [ "https://nix-qchem.cachix.org" "https://cache.nixos.org" ];
```

Comment:
Downside: if you have local cache, it will be ignored from here on?

Reply:
But I have to admit I have no idea how Hydra evaluations would be affected. Possibly hydra decides to use these substituters and ignore your local cache? But as soon as Hydra has evaluated sucessfully, the store of the hydra machine would have those paths anyway. So I would assume that for hydra it is of less importance?

Reply:
Why do we need `https://cache.nixos.org` in the list then? Here we can safely assume that this is part of the system configuration.

Reply:
When you accept the cache settings in the initial flake evaluation it will drop your local cache, yes. If you don't, you will use your system cache settings. But I understand that this can be unwanted. Digging in the forum and on github, it seems that also `extra-substituters` is supported. I will push a version with those. That hopefully resolves this concern.

Reply:
Version with `extra-substituters` pushed and working.